tests/Api/Responses/Expected/admin/order/get_order_item_adjustments_response.json

```diff
@@ -0,0 +1,12 @@
+{
+    "@context": "\/api\/v2\/contexts\/Adjustment",
+    "@id": "\/api\/v2\/admin\/order-items\/@integer@\/adjustments",
+    "@type": "hydra:Collection",
+    "hydra:member": [
+        {
+            "@id": "\/api\/v2\/admin\/adjustments\/@integer@",
+            "@type": "Adjustment"
+        }
```

Comment:
Suggestion:  Let's add `sylius:admin:order_item:index` to the properties of Adjustment. WDYT?